packages/cli/src/commands/governance/approve.ts

```diff
@@ -67,11 +67,11 @@ export default class Approve extends BaseCommand {
 
       await checkBuilder
         .proposalExists(id)
-        .proposalInStage(
+        .proposalInStages(
           id,
           governanceVersion.storage === '1' && Number(governanceVersion.major) < 3
-            ? 'Approval'
-            : 'Referendum'
+            ? ['Approval']
```

Comment:
do we actually need this anymore? governance verision that is out is past this version. i think the Approval stage can be considered not relevant anymore